images/dockerfiles/publish_nginx_images.sh

```diff
@@ -1,71 +1,97 @@
 #!/usr/bin/env bash
 
 set -o nounset
+set -o errexit
 
-DEV_ROLE_ARN="arn:aws:iam::760097843905:role/platform-developer"
+AWS_ACCOUNT_ID=760097843905
+AWS_PROFILE=platform-developer
+DEV_ROLE_ARN="arn:aws:iam::$AWS_ACCOUNT_ID:role/platform-developer"
```

Comment:
Could you use `$AWS_PROFILE` in the DEV_ROLE_ARN instead of `platform-developer`

Reply:
I think `DEV_ROLE_ARN` is now unused, so i'll remove that line entirely.